.github/workflows/python-publish.yml

```diff
@@ -6,26 +6,53 @@ name: Upload Python Package
 on:
   release:
     types: [created]
+  workflow_dispatch:
 
 jobs:
-  deploy:
+  deploy-unix:
+    runs-on: "ubuntu-latest"
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel twine Cython numpy
+          python setup.py install
+      - name: Build and publish
+        env:
+          TWINE_USERNAME: ${{ secrets.PYPI_USERNAME }}
+          TWINE_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+        run: |
+          python setup.py sdist
```

Comment:
If possible, I would try to release source distributions and wheel distributions at the same time. I think you could generate source dists and wheels on one platform (e.g. Linux) and then only upload wheels for Windows/other platforms. Here are some example scripts that might be useful.

https://github.com/glotzerlab/freud/blob/master/.circleci/deploy-linux.sh
https://github.com/glotzerlab/signac/blob/master/.circleci/deploy.bash

Reply:
It looks like pypi doesn't support wheels for linux without using the `manylinux` package.

Reply:
Yeah, there are compatibility requirements to make things work on the majority of Linux distros. You can use one of the manylinux container images for deployment if you want.

Reply:
Previous versions didn't have linux wheels, so I'm fine passing on it right now.